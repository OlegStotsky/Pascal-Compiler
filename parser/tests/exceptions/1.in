var
a : array[1..10] of sgasg;
