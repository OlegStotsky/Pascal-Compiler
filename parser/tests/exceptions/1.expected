Error at line 1, column 20 : couldn't find symbol sgasg
